cfme/tests/candu/test_graph_groupbytag.py

```diff
@@ -12,40 +13,49 @@
     test_requirements.c_and_u
 ]
 
-HOST_GRAPHS = ['host_cpu',
-               'host_memory',
-               'host_disk',
-               'host_network',
-               'host_cpu_state']
+GRAPHS = ['cpu',
+          'memory',
+          'disk',
+          'network',
+          'cpu_state']
 
 INTERVAL = ['Hourly', 'Daily']
 
 GROUP_BY = ['VM Location']
 
 CANDU_VM = 'cu-24x7'
 
+ENTITY = ['host', 'cluster']
+
 
 @pytest.fixture(scope='function')
-def host(temp_appliance_extended_db):
+def entity_object(temp_appliance_extended_db, entity):
     vm = temp_appliance_extended_db.rest_api.collections.vms.get(name=CANDU_VM)
-
-    vm_host = vm.host.name
-    return temp_appliance_extended_db.collections.hosts.instantiate(name=vm_host)
+    if entity == 'host':
+        vm_host = vm.host.name
+        return temp_appliance_extended_db.collections.hosts.instantiate(name=vm_host)
+    elif entity == 'cluster':
+        provider = temp_appliance_extended_db.rest_api.collections.providers.get(id=vm.ems_id)
```

Comment:
Required:
we have property for getting `custer` for specific `vm`

https://github.com/ManageIQ/integration_tests/blob/af5e08f6d1b43e2f1db7dfaa9a5c1bce0f641958/cfme/infrastructure/virtual_machines.py#L985

Reply:
@nachandr when we were looking at this the other day, we were using the entity object that was returned by the rest api. What @digitronik is suggesting here is to actually use the vm and cluster collections (this will use the GUI).

`In[17]: vm = appliance.collections.infra_vms.instantiate(name='cu-24x7', provider=vmwaretest)
In[18]: vm
Out[18]: InfraVm(name='cu-24x7', provider=VMwareProvider(endpoints={'default': <cfme.infrastructure.provider.virtualcenter.VirtualCenterEndpoint object at 0x7fdff3065650>}, name='vSphere 6.5 (nested)', key='vsphere65-nested', zone='default', start_ip='10.8.196.40', end_ip='10.8.196.40', provider_data=None), template_name=None)
In[19]: vm.cluster
Out[19]: Cluster(name=u'Cluster', provider=VMwareProvider(endpoints={'default': <cfme.infrastructure.provider.virtualcenter.VirtualCenterEndpoint object at 0x7fdff3065650>}, name='vSphere 6.5 (nested)', key='vsphere65-nested', zone='default', start_ip='10.8.196.40', end_ip='10.8.196.40', provider_data=None))
```
Nikhil and I did chat offline and this is slower with our current implementation.  I think both ways work here.